tools/bin/gke-kube-acceptance-test/acceptance_test_kube_gke.sh

```diff
@@ -8,65 +8,67 @@ assert_root
 kubectl --help
 
 NAMESPACE=$(openssl rand -hex 12)
-echo "Namespace" $NAMESPACE
+echo "Namespace $NAMESPACE"
 
 TAG=$(openssl rand -hex 12)
-echo "Tag" $TAG
+echo "Tag $TAG"
 
-docker login -u airbytebot -p $DOCKER_PASSWORD
+docker login -u airbytebot -p "$DOCKER_PASSWORD"
 VERSION=$TAG ./gradlew build
 VERSION=$TAG docker-compose -f docker-compose.build.yaml push
 
 # For running on Mac
 #sed -i .bak 's/default/'$NAMESPACE'/g' kube/overlays/dev/kustomization.yaml
 #sed -i .bak 's/dev/'$TAG'/g' kube/overlays/dev/kustomization.yaml
 
-sed -i 's/default/'$NAMESPACE'/g' kube/overlays/dev/kustomization.yaml
-sed -i 's/dev/'$TAG'/g' kube/overlays/dev/kustomization.yaml
+sed -i "s/default/$NAMESPACE/g" kube/overlays/dev/kustomization.yaml
+sed -i "s/dev/$TAG/g" kube/overlays/dev/kustomization.yaml
 
-kubectl create namespace $NAMESPACE
-kubectl config set-context --current --namespace=$NAMESPACE
+kubectl create namespace "$NAMESPACE"
+kubectl config set-context --current --namespace="$NAMESPACE"
 
-kubectl apply -k kube/overlays/dev --namespace=$NAMESPACE
-kubectl apply -f tools/bin/gke-kube-acceptance-test/postgres-source.yaml --namespace=$NAMESPACE
-kubectl apply -f tools/bin/gke-kube-acceptance-test/postgres-destination.yaml --namespace=$NAMESPACE
+kubectl apply -k kube/overlays/dev --namespace="$NAMESPACE"
+kubectl apply -f tools/bin/gke-kube-acceptance-test/postgres-source.yaml --namespace="$NAMESPACE"
+kubectl apply -f tools/bin/gke-kube-acceptance-test/postgres-destination.yaml --namespace="$NAMESPACE"
 
 sleep 180s
 
 function findAndDeleteTag () {
-    if [ $(curl --fail -LI --request GET 'https://hub.docker.com/v2/repositories/airbyte/'$1'/tags/'$TAG'/' --header 'Authorization: JWT '$2'' -o /dev/null -w '%{http_code}\n' -s) == "200" ];
+    local image="${1}"
+    local token="${2}"
+
+    if [ "$(curl --fail -LI --request GET "https://hub.docker.com/v2/repositories/airbyte/$image/tags/$TAG/" --header "Authorization: JWT '$token'" -o /dev/null -w '%{http_code}\n' -s)" == "200" ];
     then
-        echo "FOUND TAG" $TAG "in repository" $1 ",DELETING IT"
-        curl --request DELETE 'https://hub.docker.com/v2/repositories/airbyte/'$1'/tags/'$TAG'/' --header 'Authorization: JWT '$2'';
+        echo "FOUND TAG $TAG in repository $image, DELETING IT"
+        curl --request DELETE "https://hub.docker.com/v2/repositories/airbyte/$image/tags/$TAG/" --header "Authorization: JWT '$token'";
     else
-      echo "NOT FOUND TAG" $TAG "in repository" $1;
+      echo "NOT FOUND TAG $TAG in repository $image";
     fi
 }
 
 function cleanUpImages () {
-    TOKEN=$(curl --request POST 'https://hub.docker.com/v2/users/login/' --header 'Content-Type: application/json' --data-raw '{"username":"airbytebot","password":"'$DOCKER_PASSWORD'"}' | jq '.token')
+    TOKEN=$(curl --request POST 'https://hub.docker.com/v2/users/login/' --header 'Content-Type: application/json' --data-raw '{"username":"airbytebot","password":"'"$DOCKER_PASSWORD"'"}' | jq '.token')
```

Comment:
TODO: check this

Reply:
LOL, lots of quotes here. Has this been verified?

Reply:
Haha yeah I will double check this for sure